src/components/configure/customkey/AutocompleteKeys.tsx

```diff
@@ -20,13 +20,21 @@ const filterOptions = (
     (option: IKeymap) =>
       0 <= option.keycodeInfo.label.toLowerCase().indexOf(value)
   );
+  const matchedKeywords = options.filter(
+    (option: IKeymap) =>
+      0 <=
+      option.keycodeInfo.keywords.reduce(
+        (i, kwd) => Math.max(i, kwd.indexOf(value)),
+        -1
+      )
```

Comment:
```suggestion
      option.keycodeInfo.keywords.some((kwd) => 0 <= kwd.indexOf(value))
```

I think that we can reduce the loop count by using the `some` method instead of the `reduce` method and can represent this logic's intent more clearly.